network/src/connectivity_manager/mod.rs

```diff
@@ -493,6 +506,16 @@ where
         // Dial peers which are eligible but are neither connected nor queued for dialing in the
         // future.
         self.dial_eligible_peers(pending_dials).await;
+
+        // Sample final state once a minute
```

Comment:
This comment really should read "at most once a minute" because technically if this line of code isn't executed but every 5 minutes then this will only log once every 5 minutes. I hope that made sense.

Reply:
Fixed